memphis/memphis.py

```diff
@@ -180,14 +187,42 @@ async def producer(self, station_name, producer_name, generate_random_suffix=Fal
                 createProducerReq, indent=2).encode('utf-8')
             err_msg = await self.broker_manager.request("$memphis_producer_creations", create_producer_req_bytes)
             err_msg = err_msg.data.decode("utf-8")
-
-            if err_msg != "":
+            err_msg = json.loads(err_msg)
```

Comment:
when creating a producer you are getting the first schema update, you just checking for errors instead initialize your cache and start listening for updates